### PR TITLE
Color themes

### PR DESCRIPTION
**Pull request by briduffy**
screen shot 2018-12-17 at 10 00 54 am
screen shot 2018-12-17 at 10 02 09 am
screen shot 2018-12-17 at 10 19 58 am
screen shot 2018-12-17 at 10 20 13 am
screen shot 2018-12-17 at 10 27 40 am
screen shot 2018-12-17 at 10 29 22 am

Just trying to upload my screenshots of different color themes. The CSS framework is a lot more tricky than others that I'm familiar with.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'slim'
+gem 'materialize-sass'
```
Comment (reecebisel) [inline Gemfile:15]: I have already added a css framework through a cdn, so this isn't really doing anything

File: app/assets/stylesheets/static.scss [hunk 2]
```diff
@@ -15,3 +19,20 @@
     font-size: 1.5em;
   }
 }
+
+.button, button, input[type='button'], input[type='reset'], input[type='submit'] {
+  background-color: #C09AC1;
+  border-color:  #C09AC1;
+}
+
+a {
```
Comment (reecebisel) [inline app/assets/stylesheets/static.scss:28]: Usually you want to add classes to the these objects so we aren't changing things so broadly, in this case I can totally understand why you wanted it this way.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -5,10 +5,12 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
```
Comment (reecebisel) [inline app/views/layouts/application.html.erb:8]: good catch making your css load correctly by moving it to the end of the list.

**Review by reecebisel — COMMENTED** [review]
The feminine one is my favorite.

**Review by reecebisel — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (briduffy) [reply Gemfile:15]: I have attachment issues with what I'm familiar with... I'll get over it.

File: app/assets/stylesheets/static.scss (hunk 2 above)
Comment (briduffy) [reply app/assets/stylesheets/static.scss:28]: The app is prettttty small, so, for the sake of fluidity, it seemed okay this time around.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (briduffy) [reply app/views/layouts/application.html.erb:8]: That took more than a second to figure out, but, it was a mega relief when I did.

**Comment by briduffy** [comment]
Hahaha - feminine was my favorite too.

**Pull request closed without merging**